source/_includes/documentation-preferences.md

```diff
@@ -211,8 +213,8 @@ If selected, a session's pane, tab, or window will automatically close when the
 When a session will close, you can choose when to be prompted with a modal alert.
 #### Automatically log session input to files in:
 If enabled, every session's output will be logged to a file in the specified directory. File names are formatted as Date_Time.ProfileName.TerminalID.ProcessId.RandomNumber.log.
-#### When idle, send ASCII code
-If selected, the specified ASCII code will be transmitted every 30 seconds while nothing is happening. Don't use this unless you know what you're doing as it can have unexpected consequences.
+#### When idle, send ASCII code __ every __ seconds
```

Comment:
These docs are still for 2.1.4. The docs for 2.9.x (which will graduate to 3.0 when exiting beta) are in the `v3` branch. Please send the PR for that branch, instead.